.github/workflows/fms-additional-images-push.yaml

```diff
@@ -0,0 +1,79 @@
+name: Add FMS-HF-Tuning image to additional image list
+
+on:
+  workflow_dispatch:
+    inputs:
+      rhoai-release-version:
+        type: string
+        required: true
+        description: 'RHOAI Version number (for example: 2.11)'
+
+env:
+  REPO_NAME: rhoai-additional-images
+  REPO_OWNER: redhat-data-services
+  GITHUB_TOKEN: ${{ secrets.CODEFLARE_MACHINE_ACCOUNT_TOKEN }}
+  RHOAI_VERSION: ${{ github.event.inputs.rhoai-release-version }}
+  UPDATE_BRANCH: rhoai-${{ github.event.inputs.rhoai-release-version }}-add-fms-hf-image
+  RHOAI_BRANCH: rhoai-${{ github.event.inputs.rhoai-release-version }}
+  IMAGE_BASE: modh
+  IMAGE_REPO: fms-hf-tuning
+
+jobs:
+  append-fms-image-list:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+        with:
+          repository: 'redhat-data-services/rhoai-additional-images'
+          ref: rhoai-${{ github.event.inputs.rhoai-release-version }}
+          token: ${{ secrets.CODEFLARE_MACHINE_ACCOUNT_TOKEN }}
+
+      - name: Login to Quay.io
+        uses: redhat-actions/podman-login@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_ID }} # DEV COMMENT: Add Quay secrets to rhds/fms-hf-tuning repo
+          password: ${{ secrets.QUAY_TOKEN }}
+      
+      - name: Perform tests
+        run: | 
+          echo "Performing functional tests"
+
+      - name: Update image list
+        run: |
+          echo "Creating updated image list Pull Request"
+          git config --global user.email "138894154+codeflare-machine-account@users.noreply.github.com"
+          git config --global user.name "codeflare-machine-account"
+          git checkout -b $UPDATE_BRANCH
+          # Update the image list to include the fms-hf-tuning release image
+          export IMAGE_SHA=$(podman pull --quiet quay.io/$IMAGE_BASE/$IMAGE_REPO:release)
```

Comment:
Is there any way to do this without pulling the whole image? The image is 3.3Gb which is relatively large.